test/Table_Tests/src/In_Memory/Table_Xml_Spec.enso

```diff
@@ -4,8 +4,7 @@ from Standard.Test import all
 from Standard.Table.Errors import all
 import Standard.Base.Errors.Common.Type_Error
 
-# pretty printed xml actually makes the tests less pretty
-# so all but one tests depretty the xml before comparing
+# pretty xml makes the tests easier to read
 depretty xmlText =
     xmlText.replace (Regex.compile "\n\s*") ""
```

Comment:
I don't understand the new phrasing of this comment. This function undoes prettification, so why do we mention (seemingly unrelated information) that 'pretty xml makes tests easier to read'?

What about?
```
Pretty xml makes the tests easier to read, so this function allows to convert a pretty-formatted expected XML value into a raw value that can be compared with the generated XML.
```

Reply:
Yes that is a better comment. Thank you.